addons/web/static/tests/core/utils/objects_tests.js

```diff
@@ -1,10 +1,19 @@
 /** @odoo-module **/
 
-import { pick, shallowEqual } from "@web/core/utils/objects";
+import { omit, pick, shallowEqual } from "@web/core/utils/objects";
 
 QUnit.module("utils", () => {
     QUnit.module("Objects");
 
+    QUnit.test("omit", function (assert) {
+        assert.deepEqual(omit({}), {});
+        assert.deepEqual(omit({}, "a"), {});
+        assert.deepEqual(omit({"a": "a"}, {}), {"a": "a"});
+        assert.deepEqual(omit({"a": "a"}, "a"), {});
+        assert.deepEqual(omit({"a": "a", "b": "b"}, "c", "a"), {"b": "b"});
+        assert.deepEqual(omit({"a": "a", "b": "b"}, "b", "c"), {"a": "a"});
```

Comment:
Could test with key different from value. Otherwise it is not that clear if you are key or value based.